lib/trader.js

```diff
@@ -210,6 +210,9 @@ function stateChange (state, isIdle) {
   }).catch(() => {})
 }
 
+// At machine startup two stateChanges occur virtually at the same time: 'idleState' and 'chooseCoin'
+// It was frequently seen on the server requests arriving out of order, the throttle mitigates this issue
+// This is particularlly important at startup because of the 'machine stuck' notification
```

Comment:
I'm not convinced this is the best way to handle this. We don't want to throw away state change information, this could be confusing when debugging. Are we ever seeing these events out of order on l-m? If it's just on l-s, we should be able to solve that with a counter.